src/app/api/series/popular/route.ts

```
@@ -0,0 +1,30 @@
+  try {
+    const response = await fetch(
+      `${BASE_URL}/tv/popular?api_key=${TMDB_API_KEY}&language=en-US&append_to_response=overview`,
+      { next: { revalidate: 60 * 60 * 6 } } // Cache for 6 hours
+    );
+
+    if (!response.ok) {
+      throw new Error('Failed to fetch data');
+    }
+
+    const data = await response.json();
+    const results = data.results.map((show: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)